polonius-engine/src/facts.rs

```diff
@@ -62,6 +62,14 @@ pub struct AllFacts<T: FactTypes> {
     /// `path_accessed_at(path, point)` when the `path` was accessed at point
     /// `point`. The same logic as for `initialized_at` and `moved_out_at` applies.
     pub path_accessed_at: Vec<(T::Path, T::Point)>,
+
+    /// `known_subset(origin1, origin2)` when the "placeholder" `origin1` is a known subset
+    /// of placeholder `origin2`.
```

Comment:
This comment could be a bit better. I would say something like:

---

These reflect the `'a: 'b` relations that are either declared by the user on function declarations or which are inferred via implied bounds. For example, `fn foo<'a, 'b: 'a, 'c>(x: &'c &'a u32)` would have two entries, one for `'b: 'a` and one for `'a: 'c` (note that the transitive relation `'b: 'c` is not necessarily included explicitly, but rather inferred by polonius).